scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -158,14 +158,36 @@ workflow GvsCreateFilterSet {
         preemptible_tries = 3,
     }
 
-    call PopulateFilterSetInfo {
+    # These calls to SelectVariants are being added for two reasons
+    # 1) The snps_variant_scored_vcf and indels_variant_scored_vcf output by JointVcfFiltering contains ALL variants,
+    #     but are currently ONLY annotating SNPs and INDELs respectively.
+    # 2) Those output VCFs also contain filtered sites which we don't want to put into the filter_set_info_vqsr_lite table.
```

Comment:
what does "filtered sites" here mean?

Reply:
Sites in the VCF that have the filter field set to anything other than '.' or 'PASS'.